### ISSUE TITLE
[enterprise-4.4] Issue in file updating/updating-restricted-network-cluster.adoc

### ISSUE THREAD

**Issue by pdfruth** [openshift/openshift-docs#24814]
### Which section(s) is the issue in?
https://docs.openshift.com/container-platform/4.4/updating/updating-restricted-network-cluster.html#update-mirror-repository_updating-restricted-network-cluster

The same also applies to the equivalent page in the Openshift 4.5 documentation at;
https://docs.openshift.com/container-platform/4.5/updating/updating-restricted-network-cluster.html#update-mirror-repository_updating-restricted-network-cluster

### What needs fixing?
There are a number of environment variable setting recommendations on this page.
One in particular is the `REMOVABLE_MEDIA_PATH=<path>` variable.

The subsequent `oc adm release mirror ....` command that uses this environment variable will create a directory structure beneath that path.  In the course of doing what it does, the `oc` command will create some symbolic links in that directory structure, and those symbolic links will not be created correctly if the REMOVABLE_MEDIA_PATH environment variable refers to a relative path.

Thus, the instructions for setting this variable should include guidance to inform the user that value for `<path>` should an explicit full path (starting with a slash '/').

In other words.
The following is valid
REMOVABLE_MEDIA_PATH=/mnt/usb/removeable
The following is not valid
REMOVABLE_MEDIA_PATH=./removeable

Furthermore, it is also critical to mention that whatever path is specified, that same explicit full path must be used in step;

> iii. Take the media to the restricted network environment and upload the images to the local container registry.

when the `oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror 'file://openshift/release:${OCP_RELEASE}*' ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}` command is run
